src/content/concepts/plugins.mdx

```diff
@@ -13,7 +13,7 @@ contributors:
 
 **Plugins** are the [backbone](https://github.com/webpack/tapable) of webpack. webpack itself is built on the **same plugin system** that you use in your webpack configuration!
 
-They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do.
+They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do. webpack provides [many such plugins](/plugins/) out of the box.
```

Comment:
```suggestion
They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do. Webpack provides [many such plugins](/plugins/) out of the box.
```

Reply:
Hi @chenxsan, per [the typesetting guide](https://webpack-js-org-git-fork-scott-ln-patch-1-webpack-docs.vercel.app/contribute/writers-guide/#typesetting), "webpack should always be written in lower-case letters. Even at the beginning of a sentence."

@snitin315 Yes, done.

Reply:
Wait, no. The typesetting guide links to [the media guide](https://github.com/webpack/media#name) which says "Webpack can be written with a capital W at the beginning of a sentence"! Which is it? 😁

Reply:
The one in typesetting guide is outdated. I've file a pull request to fix it https://github.com/webpack/webpack.js.org/pull/5358

Reply:
Great, thanks. There was another one in the preceding paragraph so I've just changed that too.